worker/handlers/trigger.js

```diff
@@ -0,0 +1,39 @@
+const redis = require('redis');
+const Joi = require('joi');
+
+const redisConfig = require('../config');
+const logger = require('../../logger');
+const { channels } = require('../index');
+
+function onTrigger(message) {
+  if (!message) {
+    logger.info('Misisng query parameter!');
+    process.exit(1);
+  }
+
+  const client = redis.createClient(redisConfig);
```

Comment:
Please create this client (e.g. as triggerPbulisher) in the index.js like the rest, then use  `on subscribe` event of the repository subsciber to see if the channel is listening. You need to publish in the callback.